ci/upload_to_owncloud.sh

```diff
@@ -53,11 +53,15 @@ main() {
 
     info "upload $local_path to $OWNCLOUD_BROWSE_URL/$remote_path/$(basename "$local_path") (using user $user)"
     if ! find "$local_path" -type d -exec \
-            realpath {} --relative-to="$(dirname "$local_path")" \; | \
+            realpath {} --relative-to="$(dirname "$local_path")" \; | {
+                error=0
                 while read -r -s dir; do
                     printf . # show progress
                     dbg "Create directory: $dir"
-                    curl ${QUIET:+ -s -S} -f -n -X MKCOL "$OWNCLOUD_URL/$user/$remote_path/$dir"
+                    curl ${QUIET:+ -s -S} -f -n -X MKCOL "$OWNCLOUD_URL/$user/$remote_path/$dir" || {
+                        error="$?"
+                        err "Failed to create dir: $remote_path/$dir/ (error $error)"
```

Comment:
Does it make sense to continue this iteration if the directory creation did not succeed?